.travis.yml

```diff
@@ -77,6 +77,7 @@ before_install:
   - pushd builds
   # Build into own virtualenv
   # We therefore control our own environment, avoid travis' numpy
+  - pip install -U 'virtualenv==14.0.4'
```

Comment:
Should probably add a comment as to why the version is pinned. We will probably want to revisit this when the next venv shows up ;)